Tests/TensorFlowTests/InitializerTests.swift

```diff
@@ -37,6 +37,32 @@ final class InitializerTests: XCTestCase {
         XCTAssertEqual(ShapedArray(
             shape: [4, 3],
             scalars: [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0]), x.array)
+
+        let linearSpaceTensor1 = Tensor<Float>(linearSpaceFrom: 0.1, to: 2.0, count: 10)
+        assertEqual(
+            Tensor<Float>(
+                [
+                    0.1, 0.31111111, 0.52222222, 0.73333333, 0.94444444, 1.15555555, 1.36666666,
+                    1.57777777, 1.78888888, 2.0
+                ]), linearSpaceTensor1, accuracy: 0.001)
+
+        let linearSpaceTensor2 = Tensor<Float>(linearSpaceFrom: 2.0, to: 0.1, count: 10)
+        assertEqual(
+            Tensor<Float>(
+                [
+                    2, 1.78888889, 1.57777778, 1.36666667, 1.15555556, 0.94444444, 0.73333333,
+                    0.52222222, 0.31111111, 0.1
+                ]), linearSpaceTensor2, accuracy: 0.001)
+
+        let linearSpaceTensor3 = Tensor<Float>(linearSpaceFrom: -5.0, to: 8.5, count: 20)
+        assertEqual(
+            Tensor<Float>(
```

Comment:
This call to `init(_:)` may not be necessary because `Tensor` is expressible by array literal.

Reply:
Good call, that simplifies things. Fixed that.